### PR TITLE
Completed css-layout-classes exercise - Joe Dei Rossi

### PR DESCRIPTION
**Pull request by JoeDeiRossi**
image

### CODE REVIEW

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
Hey Joe, nice start, but you want to have columns that reflect the sizes in the example, particularly a 1/3 and 2/3 class. you will notice the text on the first line and the images are not in thirds. also the text for mario and donkey kong text should be 2/3. Nice work on the HTML!

**Comment by JoeDeiRossi** [comment]
image

File: css-layout-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <header>
+          <h1>Choose your character!</h1>
+        </header>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-quarter">
```
Comment (BrettAlbright) [inline css-layout-classes/index.html]: just need to update these all to 1/3 columns

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
almost there, one more update!

**Comment by JoeDeiRossi** [comment]
Third time's the charm!

image

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**